Expensify/App
# Display Chat Reports

cc @tgolen ~~Simply just gets the chat IDs and shows them in the LHN. The titles all still say "Chat Report" for now.~~

Holding on the deploy of https://github.com/Expensify/Web-Expensify/pull/29003

Fixes https://github.com/Expensify/ReactNativeChat/issues/268

Updated this one to have the full report titles.

# Tests

1. Create some chat reports by calling `API.createChatReport({emailList: <some csv email list>})`
1. Login to ChatDot and verify that the chat reports load and have correct titles

<img width="1435" alt="Screenshot 1274" src="https://user-images.githubusercontent.com/32969087/91371026-cd2b6180-e7c4-11ea-878b-c2f29fc477c5.png">
